lldb/test/API/lang/swift/expression/generic_function_name/TestSwiftGenericFunctionName.py

```diff
@@ -0,0 +1,35 @@
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+import unittest2
+
+
+class TestSwiftGenericFunction(lldbtest.TestBase):
+    @swiftTest
+    def test(self):
+        """Test display of generic function names"""
+        self.build()
+        target, process, thread, bkpt = lldbutil.run_to_name_breakpoint(
+            self, 'f')
+
+        # Live process:
+        stream = lldb.SBStream()
+        self.frame().GetDescription(stream)
+        desc = stream.GetData()
+        # It's debatable whether C's generic parameter should be displayed here.
+        self.assertIn("C.f<Int>(t=1, u=2)", desc)
+
+        # Dead process + debug info:
+        process.Kill()
+        stream = lldb.SBStream()
+        bkpt.GetLocationAtIndex(0).GetDescription(stream, 1)
+        desc = stream.GetData()
+        self.assertIn("C.f<T>(T, U) -> ()", desc)
```

Comment:
shouldn't this be `C.f<U>(T, U)`? The function's generic parameter is named `U` in the source, and its containing class has the generic type named `T`. If this test is passing, do we have a bug?

Reply:
Uh, thanks!

Reply:
I'll track that separately and fix it in a follow-up, since it isn't a regression.